drivers/iommu/riscv/iommu.h

```diff
@@ -145,7 +173,7 @@ static inline void riscv_iommu_writeq(struct riscv_iommu_device *iommu,
 	writeq_relaxed(val, iommu->reg + offset);
 }
 
-int riscv_iommu_probe(struct device *dev, phys_addr_t reg_phys, size_t reg_size);
+int riscv_iommu_init_common(struct riscv_iommu_device *iommu);
```

Comment:
Why not `riscv_iommu_probe_common()`?

Reply:
We are done with probing before calling that function, we've already mapped the iommu's register space and got the irqs assigned.